go.mod

```diff
@@ -1,3 +1,17 @@
 module code.vegaprotocol.io/shared
 
 go 1.17
+
+require (
+	github.com/adrg/xdg v0.3.3
+	github.com/stretchr/testify v1.7.0
+	github.com/zannen/toml v0.3.2
+	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/sys v0.0.0-20210615035016-665e8c7367d1 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
+)
```

Comment:
Why 2 require block?

Reply:
I think it's go 1.17 to separate direct and indirect

Reply:
Ref https://golang.org/doc/go1.17#go-command

>Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.

Reply:
Is suppose it's compatible < 1.16  to anyway.